libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/TreatmentView/TreatmentList.tsx

```diff
@@ -92,4 +99,14 @@ export class TreatmentList extends React.Component<ITreatmentListProps> {
       </div>
     );
   }
+  private toScientific(item: { [key: string]: any }): { [key: string]: any } {
+    const copy = _.cloneDeep(item);
+    Object.keys(copy).forEach((k) => {
+      const isFloat = typeof copy[k] === "number" && copy[k] % 1 !== 0;
+      if (isFloat) {
+        copy[k] = copy[k].toExponential(3);
+      }
+    });
+    return copy;
+  }
```

Comment:
this one seems very useful, probably move to util in core pkg could be very helpful.